tests/test_index.py

```diff
@@ -866,38 +934,76 @@ def test_patch_from_tarball(testing_workdir):
 
     assert "revoke_test-1.0-0.tar.bz2" in pkg_list
     assert "zlib" in pkg_list["revoke_test-1.0-0.tar.bz2"]["depends"]
-    assert "package_has_been_revoked" not in pkg_list["revoke_test-1.0-0.tar.bz2"]["depends"]
+    assert (
+        "package_has_been_revoked"
+        not in pkg_list["revoke_test-1.0-0.tar.bz2"]["depends"]
+    )
 
     assert "remove_test-1.0-0.tar.bz2" in pkg_list
 
 
 def test_index_of_removed_pkg(testing_metadata):
-    out_files = api.build(testing_metadata)
-    for f in out_files:
+
+    archive_name = "test_index_of_removed_pkg-1.0-1.tar.bz2"
+    archive_destination = os.path.join(
+        testing_metadata.config.croot, TEST_SUBDIR, archive_name
+    )
+
+    # conda_build.api.build() calls update_index as a side effect.
+    # used to build on every test (38 seconds)
```

Comment:
this is what we are avoiding

Reply:
Perhaps this information would be better in a doc string? If you can't tell by now I really dislike leaving old code around that is commented out 😅

Reply:
Fixed